proto/gubernator.proto

```diff
@@ -159,6 +159,11 @@ message RateLimitReq {
 
   // Maximum burst size that the limit can accept.
   int64 burst = 8;
+
+  // This is metadata that is associated with this rate limit. Peer to Peer communication will use
+  // this to pass trace context to other peers. Might be useful for future clients to pass along
+  // trace information to gubernator.
+  map<string, string> metadata = 9;
```

Comment:
Suggestion: OTel calls this data a "trace carrier".  "metadata" has a different meaning in this context.

Reply:
Originally I called it that, but decided there might be a future need to send additional metadata. So why limit ourselves by giving it a specific name. The tracing propagators will only grab the connect values it's interested in, which leaves us free to add other values in the future.

Reply:
Fair enough.